dotCMS/src/integration-test/java/com/dotcms/util/pagination/ContentTypesPaginatorTest.java

```diff
@@ -121,6 +126,38 @@ public void test_getItems_WhenFilterContainsContentTypeName_ReturnsTheContentTyp
         }
     }
 
+    /**
+     * Method to Test: {@link ContentTypesPaginator#getItems(User, String, int, int, String, OrderDirection, Map)}
+     * When: When the current indices are desactive or deleted
+     * Should: Should return NA in the entries property
+     *
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @Test
+    public void whenTheIndicesAreDeactivateShouldReturnNAInEntries() throws DotDataException, IOException {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5857184)